product_docs/docs/edbcloud/beta/overview/03_security_compliance.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: "Security"
+---
+
+EDB's cloud offering runs in your own cloud account, isolates your data from other users, and gives you control over our access to it. The key security features are:
+- **Data Isolation:** Clusters are installed and managed in your cloud environment. Complete segregation of your data is ensured: your data is never stored in a shared space, and compromise of another EDB customer's system does not put your system at risk.
+- **Granular Access Control:** You can use _Single Sign On_ and define your own sets of _Roles_ and _Role Based Access Control_ policies to manage your individual cloud environments.
+- **Data Encryption:** Data is encrypted in motion and at rest. All network traffic is encrypted using Transport Layer Security (TLS). All data is encrypted when stored in database clusters. 
+- **Control Plane Logging:** Activities in the control plan related to User Roles, Organization updates, and Cluster Creation and Deletion are tracked automatically and viewed in the Activity Log.   
+- **Database Logging and Auditing:** Creation and deletion of clusters are tracked automatically in the Activity Log and PostgreSQL logging, and auditing is supported. Functionality to track and analyze database activities is enabled automatically. The PostgreSQL Audit Extension (pgAudit) is enabled automatically for you when deploying a Postgres cluster. For EDB Postgres Advanced, the EDB Audit extension is enabled automatically for you. 
```

Comment:
```suggestion
- **Database Logging and Auditing:** Functionality to track and analyze database activities is enabled automatically. The PostgreSQL Audit Extension (pgAudit) is enabled automatically for you when deploying a Postgres cluster. For EDB Postgres Advanced, the EDB Audit extension is enabled automatically for you. 
```

Reply:
Can we sub-bullet pgaudit and edb-audit and document how they're configured? And where the audit logs go to?

Reply:
Yes, I'm just waiting for the `edb_admin` permissions merge (pg_minotiro) so that I can verify EDB Audit settings.

Reply:
> I didn't see all the changes in the Database Logging and Auditing section captured in the latest version of the topic. I made a few changes that didn't seem to stick in #1692.